### ISSUE TITLE
Android mobile app is not working 2.3.1

### ISSUE THREAD

**Issue by leonidmamontov** [RocketChat/Rocket.Chat#3845]
Your Rocket.Chat version: 0.35.0
The  problem is: abort, abort message when i try to connect to server.
Mobile Android 5.1

Chat works on desktop and any browsers, but not working on Android mobile.
Yes, i try to remove (full remove) application and install again - no any effect.

**Comment by Artharius** [comment]
same! with the https (using SSL Comodo) my android app does not connect to server !. From the web browser I can.

**Comment by TheReal1604** [comment]
@leonidmamontov @Artharius I know this issue if something is wrong with your ssl cert (or the device does not trust them) Can you please check that your android devices trust your ssl cert on the server? For my Users all is working fine with a Lets Encrypt Cert and the Rocket.Chat Play Store App.

**Comment by leonidmamontov** [comment]
I use WoSign
https://www.ssllabs.com/ssltest/analyze.html?d=rc.365works.ru
I also know about Extra download but i don't know how to fix it.
Can you help with it?

**Comment by TheReal1604** [comment]
Eventually, what you use as your reverseproxy? Apache2?

Then you have to get the intermediate certificate (i think from here: https://www.wosign.com/English/root.htm) and change your config for ssl certs as follow:

`SSLEngine on
SSLCertificateFile /path/to/your_domain_name.crt
SSLCertificateKeyFile /path/to/your_private.key
SSLCertificateChainFile /path/to/Intermediate.crt`

SSLLabs worries about your missing intermediate certificate, this must be sent from your webserver.

Hope i can help you with this steps...

**Comment by leonidmamontov** [comment]
I get 2_rc.365works.ru.crt and 1_root_bundle.crt

**Comment by leonidmamontov** [comment]
# HTTPS Server
upstream backend {
    server 127.0.0.1:3000;
}
    server {
        listen 443 ssl;
        server_name rc.365works.ru;
        error_log /var/log/nginx/rocketchat_error.log;
        ssl_certificate /etc/nginx/certificate.crt;
        ssl_certificate_key /etc/nginx/certificate.key;
        ssl_dhparam /etc/nginx/dhparams.pem;
        ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
        ssl_ciphers 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:ECDHE-RSA-DES-CBC3-SHA:ECDHE-ECDSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:AES:DES-CBC3-SHA:HIGH:SEED:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!RSAPSK:!aDH:!aECDH:!EDH-DSS-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA:!SRP';
        ssl_prefer_server_ciphers on;
        ssl_session_cache shared:SSL:50m;
        ssl_session_timeout 1d;
        ssl_session_tickets off;
        add_header Strict-Transport-Security "max-age=63072000";

        location / {
            proxy_pass http://ip:3000/;
            proxy_http_version 1.1;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection "upgrade";
            proxy_set_header Host $http_host;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forward-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forward-Proto http;
            proxy_set_header X-Nginx-Proxy true;
            proxy_redirect off;
        }
    }

**Comment by TheReal1604** [comment]
I think then you follow this tutorial from digicert, you find the right way ;-). Hint: You have to create a bundle - not like apache2.

https://www.digicert.com/ssl-certificate-installation-nginx.htm

**Comment by TheReal1604** [comment]
@leonidmamontov Do you have success?

**Comment by Artharius** [comment]
Hello everyone! apologize for not answering before but have been busy days. I found a curious solution to the problem ... It turns out that if you try to enter for the first time via app with wifi since this tells me "failed, abort abort", which left me thinking that one day out of the reach of Wi-Fi signal and the 3G- or 4G LTE enabled, it worked !!. Then configuring smartphones office staff for the first time I did it only with 3G and perfect.

If you want to enter the server via smartphone app use 3g for the first time in instead wifi and all good. I hope it works to you too. Greetings.
